src/MySql/Sql/CreateAuditTrigger.php

```diff
@@ -158,39 +165,75 @@ public function buildStatement()
         throw new FallenException('action', $this->triggerAction);
     }
 
-    $sql = sprintf('
-create trigger `%s`.`%s`
-after %s on `%s`.`%s`
-for each row
-begin
-',
-                   $this->dataSchemaName,
-                   $this->triggerName,
-                   $this->triggerAction,
-                   $this->dataSchemaName,
-                   $this->tableName);
+    $sql   = [];
+    $sql[] = sprintf('create trigger `%s`.`%s`', $this->dataSchemaName, $this->triggerName);
+    $sql[] = sprintf('after %s on `%s`.`%s`', $this->triggerAction, $this->dataSchemaName, $this->tableName);
+    $sql[] = 'for each row';
+    $sql[] = 'begin';
 
-    $sql .= $this->skipStatement($this->skipVariable);
+    $sql = $this->skipStatement($sql, $this->skipVariable);
 
     if (is_array($this->additionalSql))
     {
       foreach ($this->additionalSql as $line)
       {
-        $sql .= $line;
-        $sql .= "\n";
+        $sql[] = $line;
       }
     }
 
-    $sql .= $this->createInsertStatement($rowState[0]);
+    $sql[] = $this->createInsertStatement($rowState[0]);
     if (sizeof($rowState)==2)
     {
-      $sql .= $this->createInsertStatement($rowState[1]);
+      $sql[] = $this->createInsertStatement($rowState[1]);
     }
 
-    $sql .= isset($this->skipVariable) ? "end if;\n" : '';
-    $sql .= 'end';
+    $sql[] = isset($this->skipVariable) ? "end if;" : '';
+    $sql[] = 'end';
+
+    return $this->writeIndent($sql);
+  }
+
+  //--------------------------------------------------------------------------------------------------------------------
+  /**
+   * Returns the SQL statement with indents.
+   *
+   * @param string[] $sqlStatement The SQL query.
+   *
+   * @return string
+   */
+  private function writeIndent($sqlStatement)
+  {
+    $sqlStatementWithIndent = [];
+    foreach ($sqlStatement as $key => $line)
+    {
+      $words = explode(' ', $sqlStatement[$key]);
```

Comment:
* Use $line instead of $sqlStatement[$key].
* Trim line before using it. Just incase a line as leading white space.